contrib/cirrus/mac_cleanup.sh

```diff
@@ -15,10 +15,15 @@ killall podman vfkit gvproxy make go ginkgo
 # "short" as sockets may reside here.  Darwin suffers from
 # the same limited socket-pathname character-length restriction
 # as Linux.
+chmod -R u+w /private/tmp/ci/* /private/tmp/ci/.??*
 rm -rf /private/tmp/ci/* /private/tmp/ci/.??*
 
-# Don't clobber the $CIRRUS_WORKING_DIR for this (running) task.
+# Don't change or clobber anything under $CIRRUS_WORKING_DIR for
+# the currently running task.  But make sure we have write permission
+# (go get sets dependencies ro) for everything else, before removing it.
 # shellcheck disable=SC2154
+find "${ORIGINAL_HOME:-$HOME}/ci" -mindepth 1 -maxdepth 1 \
+    -not -name "*task-${CIRRUS_TASK_ID}*" -prune -exec chmod -R u+w '{}' +
```

Comment:
Are you sure this is a `go get` thing, and not a namespace thing? Are any of the failed deletes happening under `.local/share/containers` or something like that? If they are, this is not going to help, you're going to need `podman unshare`.

Reply:
:smile: was _just_ writing up a comment about that.  See the 'prep' drop-down [under this run](https://cirrus-ci.com/task/6400831147212800?logs=prep#L1).